Python/pywarpx/picmi.py

```diff
@@ -2241,11 +2241,13 @@ def diagnostic_initialize_inputs(self):
             E_fields_list = ['Er', 'Et', 'Ez']
             B_fields_list = ['Br', 'Bt', 'Bz']
             J_fields_list = ['Jr', 'Jt', 'Jz']
+            Jdisp_fields_list = ['Jr_disp', 'Jt_disp', 'Jz_disp']
```

Comment:
I think it would be better to call this `J_displacement_field_list`. It is longer but much clearer (when I see disp I think dispersion).